sklearn/ensemble/forest.py

```diff
@@ -861,7 +861,7 @@ class RandomForestClassifier(ForestClassifier):
         by `np.random`.
 
     verbose : int, optional (default=0)
-        Controls the verbosity of the tree building process.
+        Controls the verbosity of both `fit` and `predict`.
```

Comment:
This will also influence other funcs like predict_proba. Maybe use terms like ``when fitting and predicting``?

Reply:
Yeah, that makes sense.